packages/deployment/upgrade-test/Dockerfile

```diff
@@ -1,27 +1,20 @@
 # Defaults
-ARG BASE_IMAGE=ghcr.io/agoric/agoric-3-proposals@sha256:ac10c09b5927d759d37b1525b5a0bc4aeb3034df8d0f0fbb15b51203555ffb1b
+ARG BASE_IMAGE=ghcr.io/agoric/agoric-3-proposals:pr33
```

Comment:
`:main` isn't on target for this?

`pr33`, while more readable than a hash, seems somewhat fragile. "pr33" in a comment and a hash in the code would make more sense to me.

_not critical_